src/CoreWCF.CrankTests/CoreWCFPerfService/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using CoreWCF;
+using CoreWCF.Configuration;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+
+namespace CoreWCFPerfService
+{
+    public class Parameters
+    {
+        public const string Url = "baseaddress";
+    }
+
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            string filePath = Path.Combine(Environment.CurrentDirectory, "CoreWCF.PerfService.exe");
+            Console.WriteLine(filePath);
+            string command = $" advfirewall firewall add rule name=\"CoreWCFPerfService\" dir=in protocol=any action=allow program=\"{filePath}\" enable=yes";
+            ExecuteCommand(command, Environment.CurrentDirectory, TimeSpan.FromSeconds(20));
```

Comment:
See my comment on the WCF client crank test PR about modifying the firewall rules programmatically.

Reply:
I used the firewall rules you recommended, but when I build the project with dotnet build command, I get the error "error MSB4803: The task "ResolveComReference" is not supported on the .NET Core version of MSBuild."